documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -295,44 +295,7 @@ In order to create a valid Integration DID message, the following steps are to b
 
 ### Delete
 
-In order to deactivate a DID document, a valid Integration DID message must be published that removes all content from a DID Document, effectively deactivating the DID Document. Keep in mind that this operation is irreversible.
-
-## IOTA Identity standards
-
-The `did:iota` method is implemented in the [IOTA Identity framework](https://github.com/iotaledger/identity.rs). This framework supports a number of operations that are standardized, some are standardized across the SSI community, and some are the invention of the IOTA Foundation. 
-
-### Standardized Verification Method Types
-
-We support two different Verification Method Types. Verification methods that can be used for signing DID Documents, and verification methods that are only used for signing [W3C standardized Verifiable Credentials](https://www.w3.org/TR/vc-data-model/). This set differs as the IOTA Identity implements revocation of Verifiable Credentials through public key removal, which requires every credential to be signed by a different keypair. We create collection of keypairs in a Merkle Tree in order to save space in the DID Document.
-
-Verification Methods that can be used to sign DID Documents and do other repeatable activities:
-* `Ed25519VerificationKey2018` to create a `JcsEd25519Signature2020`.
-
-Verification Methods that can be used to sign Verifiable Credentials:
-* `MerkleKeyCollection2021` to create a `MerkleKeySignature2021`.
-
-### Revocation
```

Comment:
Might be good to keep a simple "revoke key -> revoke credential" around, recognizing its shortcoming and recognizing that another revocation method should be supported.

Reply:
Re-added the "Revocation" section which mentions this.